src/mlpack/methods/dbscan/dbscan_main.cpp

```diff
@@ -86,26 +88,49 @@ void RunDBSCAN(RangeSearchType rs = RangeSearchType())
 
   // Load dataset.
   arma::mat dataset = std::move(CLI::GetParam<arma::mat>("input"));
-
+  
```

Comment:
Looks like we can remove the extra whitespace here.

Reply:
Oh, Thank you!!